### ISSUE TITLE
Nav bar temporarily vanishes when navigating up

### ISSUE THREAD

**Issue by ohAitch** [urbit/tree#16]
I understand the "only this node" placeholder was removed so that webtalk didn't have a flash of default nav, but could the dpad still be on until explicitly disabled? Otherwise every time you go up a step, you have to wait a couple seconds for the siblings to load before you can go up another step.

**Comment by galenwp** [comment]
Makes sense to me.  The home and dpad could stay, and the siblings could go.

**Comment by galenwp** [comment]
This is a fairly easy fix for an outside contributor.
